cob_twist_controller/include/cob_twist_controller/limiters/limiter_base.h

```diff
@@ -54,4 +77,27 @@ class LimiterBase
         const LimiterParams& limiter_params_;
```

Comment:
as the LimiterJointBase class derives from LimiterBase, they do not need their own LimiterParams...remove